diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
-```
+curl --request GEt http://www.example.com/donuts```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function() {
+$.ajax({
```

Comment:
It's good to get into the habit of prevent your default actions by passing the event and prevent the default actions so something like this 
```
let createDonut = function (event) {
  event.preventDefault();
```